### ISSUE TITLE
Please create docs for zynaddsubfx instead of yoshimi

### ISSUE THREAD

**Issue by falkTX** [ahlstromcj/yoshimi-doc#2]
As the title says, please consider doing the documentation for zynaddsubfx rather than yoshimi.

Yoshimi was created when zynaddsubfx development was not going well, and wrongly got the reputation of behaving better than zyn.
Nowadays zyn development is going strong, with 2.5.1 released just a few weeks ago.
With 3.0  there will be a completely new interface, inspired by this mockup:
http://budislavtvp.deviantart.com/art/ZynAddSubFX-UI-Concept-2014-455890191

You can read about zyn vs yoshimi history extensively here:
http://linuxmusicians.com/viewtopic.php?f=48&t=13912#p62686

In short, there's no real reason to continue with yoshimi development instead of zynaddsubfx.
So please consider making the manual for zynaddsubfx instead.

**Comment by trebmuh** [comment]
I do not know all the background of the ZynAddSubFX/Yoshimi thing as well and I might be wrong if those 2 projects have diverted deeply in the code.
That said, what's been said above makes sense to me.

**Comment by ahlstromcj** [comment]
Sorry, Filipe, you're dead wrong.  Both project should go forward as best
they can.  There is no need for a "winner".

No reason to continue with Yoshimi?  It looks better, and may well work
smoother.  Besides, it scratches the author's itch.  I'm currently into
Yoshimi.  I like the fact that the automation control for Yoshimi is
advancing.  And I'll bet the "competition" benefits both projects.

My yoshimi-doc covers ZynAddSubFX pretty well, since it overlaps a lot with
Yoshimi.  At some point, should I find time, I will add extra material to
cover the differences between the two project.  At least generate a "feature
matrix".

Any, don't mean to be gruff, but telling someone they should stop their
efforts is a bit impolite.

Thanks for the links!  The proposed UI looks cool!  Still, what counts,
really, is the engine and the automation.

Chris

-------- Filipe Coelho 09:57 Sun 26 Jul      --------

> As the title says, please consider doing the documentation for zynaddsubfx rather than yoshimi.
> 
> Yoshimi was created when zynaddsubfx development was not going well, and wrongly got the reputation of behaving better than zyn.
> Nowadays zyn development is going strong, with 2.5.1 released just a few weeks ago.
> With 3.0  there will be a completely new interface, inspired by this mockup:
> http://budislavtvp.deviantart.com/art/ZynAddSubFX-UI-Concept-2014-455890191
> 
> You can read about zyn vs yoshimi history extensively here:
> http://linuxmusicians.com/viewtopic.php?f=48&t=13912#p62686
> 
> In short, there's no real reason to continue with yoshimi development instead of zynaddsubfx.
> So please consider making the manual for zynaddsubfx instead.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/ahlstromcj/yoshimi-doc/issues/2

## 

The random quantum fluctuations of my brain are historical accidents that
happen to have decided that the concepts of dynamic scoping and lexical
scoping are orthogonal and should remain that way.
        -- Larry Wall in 199709021854.LAA12794@wall.org

**Comment by falkTX** [comment]
Sorry but I have to disagree.

ZynAddAubFX is a big project, something that takes a lot of man hours to maintain.
A lot of work went into making zyn UI and DSP sides separate, and that work is currently being wasted by continuing with yoshimi development.

I have to be honest with you here, the Yoshimi developer(s) don't know what they are doing.
The more code and effort I see being put into that project the more I feel bad about the whole thing.
The way the LV2 plugin was done is atrocious, and no one else said anything yet because:
- they don't want to be impolite
- they understand the history behind the project, so things require extra care

But well, it needs to be said. Yoshimi is a project that should be abandoned.

This is nothing personal against you, but I'm tired of the (wrong) impression that Yoshimi is somehow better than Zyn.
Or that it's good to have both - It's NOT.
The file formats already started to deviate and the more time goes on it will get worse until the 2 formats are incompatible.
The time "wasted" on Yoshimi could be very well spent on some core Zyn features instead.

Anyway, sorry for the rant.
What I ask here is for you to consider doing the documentation for Zyn rather than Yoshimi.
